package.json

```diff
@@ -53,12 +53,15 @@
     "grunt-contrib-concat": "^1.0.1",
     "grunt-contrib-connect": "^2.0.0",
     "grunt-contrib-jshint": "^1.1.0",
-    "grunt-contrib-qunit": "^1.3.0",
+    "grunt-contrib-qunit": "^10.1.1",
     "grunt-contrib-requirejs": "^1.0.0",
     "grunt-contrib-uglify": "~4.0.1",
     "grunt-contrib-watch": "~1.1.0",
     "grunt-sass": "^3.1.0",
     "jquery-mousewheel": "~3.1.13",
-    "node-sass": "^7.0.3"
+    "sass": "^1.77.8"
+  },
+  "dependencies": {
+    "sass": "^1.77.8"
```

Comment:
Fairly certain this can be removed since this is defined as a dev dependency.
```suggestion
```

Reply:
Indeed that can, thanks; this seems to have been a mistake during merge/rebase.